### PR TITLE
UI/single historical month

### PR DESCRIPTION
**Pull request by hashishaw**
When filtering on the history tab down to one month of data, the UI updates to show only the following: 

Screen Shot 2022-05-10 at 10 22 41 AM

### CODE REVIEW

**Review by hellobontempo — APPROVED** [review]
Thank you so much for tackling! Just a few minor comments/cleanup to address then good to go! 🚀

File: ui/app/templates/components/clients/history.hbs [hunk 1]
```diff
@@ -155,7 +155,7 @@
               @timestamp={{this.responseTimestamp}}
             />
           {{/if}}
-          {{#if this.byMonthActivityData}}
+          {{#if this.hasMultipleMonthsData}}
```
[the remark this answers is not in the document]
Comment (hellobontempo) [reply ui/app/templates/components/clients/history.hbs:158]: Oh right! The single month stats that show are _within_ that component. Thanks for clarifying!!

**Comment by hellobontempo** [comment]
Saw tests failed so started a rerun from CircleCi

File: ui/app/templates/components/clients/running-total.hbs [hunk 2]
```diff
@@ -1,88 +1,128 @@
...
-
-    <div class="chart-container-wide">
-      <Clients::LineChart @dataset={{@lineChartData}} @upgradeData={{@upgradeData}} />
+    <div class="single-month-stats">
+      <div class="single-month-section-title">
+        <h2 class="chart-title">New clients</h2>
+        <p class="chart-subtext">
+          This is the number of clients which were created in Vault for the first time in the selected month.
+        </p>
+        <p class="data-details">{{this.showSingleMonth.new.total}}</p>
```
Comment (hellobontempo) [inline ui/app/templates/components/clients/running-total.hbs:16]: just noticed these stats need the `format-number` helper!
```suggestion
        <p class="data-details">{{format-number this.showSingleMonth.new.total}}</p>
```

File: ui/app/templates/components/clients/running-total.hbs [hunk 3]
```diff
@@ -1,88 +1,128 @@
...
-      <p class="data-details">
-        {{format-number this.entityClientData.runningTotal}}
-      </p>
+    <div class="single-month-stats">
+      <div class="single-month-section-title">
+        <h2 class="chart-title">Total monthly clients</h2>
+        <p class="chart-subtext">
+          This is the number of total clients which used Vault for the given month, both new and previous.
+        </p>
+        <p class="data-details">{{this.showSingleMonth.total.total}}</p>
```
Comment (hellobontempo) [inline ui/app/templates/components/clients/running-total.hbs:31]:
```suggestion
        <p class="data-details">{{format-number this.showSingleMonth.total.total}}</p>
```

File: ui/mirage/handlers/clients.js [hunk 4]
```diff
@@ -256,7 +125,7 @@ const MOCK_MONTHLY_DATA = [
     },
   },
   {
-    timestamp: '2021-09-01T00:00:00Z',
+    timestamp: formatISO(startOfMonth(sub(new Date(), { months: 2 }))),
```
[the remark this answers is not in the document]
Comment (hashishaw) [reply ui/mirage/handlers/clients.js:307]: Updated the ordering of the months (from oldest to newest) and the `handleMockQuery` method

**Pull request merged**